system/lib/libc/README.md

```diff
@@ -1,4 +1,5 @@
-This folder contains the musl version of libc at `/musl`.  The upstream version can be found at http://www.musl-libc.org/
+This folder contains the musl version of libc at `/musl`. The upstream version can be found at http://www.musl-libc.org/.
+Most of the source comes from musl 1.1.15. Some is from older versions, but I'm assuming that this is because these files haven't changed between versions.
```

Comment:
Can you drop this last sentence of re-phrase.  The last update was indeed to 1.1.15 in #4813.

Reply:
Done.